akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -737,7 +737,7 @@ private[stream] final class WireTap[T] extends GraphStage[FanOutShape2[T, T, T]]
           }
         }
 
-        override def onDownstreamFinish(): Unit = {
+        override def onDownstreamFinish(cause: Throwable): Unit = {
```

Comment:
Cancellation has no effect on overall stream so :+1: to not propagating `cause`.